convert/convert/converters/pdf/pdf_converter.py

```diff
@@ -13,17 +14,17 @@
 
 
 class PlainPDFToBadgerdocTokensConverter:
-    def __init__(self):
+    def __init__(self) -> None:
         self.offset = 0
-        self.page_size = None
+        self.page_size: Any = None
 
-    def get_bbox(self, char: LTChar) -> tuple:
-        return (
-            char.x0,
-            self.page_size.height - char.y1,
-            char.x1,
-            self.page_size.height - char.y0,
-        )
+    def get_bbox(self, char: LTChar) -> List[float]:
+        return [
+            float(char.x0),
+            float(self.page_size.height - char.y1),
+            float(char.x1),
+            float(self.page_size.height - char.y0),
+        ]
```

Comment:
Could you clarify this change?

Reply:
In other parts of code bbox is defined as a list of floats

Reply:
Done